database/fakerData.js

```diff
@@ -0,0 +1,11 @@
+var faker = require('faker');
```

Comment:
Dan, did you say that you are working on getting faker to work?

Reply:
You can disregard this piece.  I didn't realize it was in this copy.  It's more just preliminary notes based on my research.  still researching.